app/src/common/shared/com/igalia/wolvic/ui/adapters/DownloadsAdapter.java

```diff
@@ -169,6 +169,7 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
                                     binding.thumbnail.setImageBitmap(bitmap);
                             }
                     );
+                    // TODO: AsyncTask has been deprecated
```

Comment:
Let's open another issue for AsyncTask changes as well please

Reply:
https://github.com/Igalia/wolvic/issues/801